components/storage/blockdevice/COMPONENT_SD/SDBlockDevice.cpp

```diff
@@ -248,6 +248,7 @@ using namespace mbed;
 
 // Only HC block size is supported. Making this a static constant reduces code size.
 const uint32_t SDBlockDevice::_block_size = BLOCK_SIZE_HC;
+const char * SDBlockDevice::_type = "SD";
```

Comment:
This goes into RAM. `const char * const` would avoid that, but would still likely waste 4 bytes on a pointer.

It should be `const char SDBlockDevice::_type[] = "SD";`. But even then, why do that? It doesn't need to be an actual variable. The `get_type()` functions can just return a string literal.

Reply:
10x. I have changed it to string literal as you proposed.